style.css

```diff
@@ -466,9 +473,341 @@ h2 {
     80% 89.5%,
     71.3% 90.3%;
   background-repeat: no-repeat;
+  border-radius: 100px 0 0;
   width: 100%;
 }
 
 .bar {
   margin: 20px 0 0;
 }
+
+.menu-list,
+.web {
+  display: none;
+}
+
+@media screen  and (min-width: 992px) and (max-width: 1100px) {
+  .menu-logo {
```

Comment:
- [ ] [OPTIONAL] It will be mush better, if you removed the `(max-width: 1100px)` so if any one open it with width more than `1100px` it looks good with him.

Reply:
ok. Have done it